docs/TROUBLE.md

```diff
@@ -0,0 +1,162 @@
+# mLRS Documentation: Diagnosing Common Problems #
+
+([back to main page](../README.md))
+
+This page attempts to provide a step by step guide to isolate and correct some common problems.  It covers only a core use case where an OpenTX or EdgeTX radio with a mLRS Tx module is used to communicate with a mLRS receiver connected to an ArduPilot flight controller to provide both RC control and MAVLink Telemetry to a ground station or [Yaapu Telemetry App](CRSF.md).  Users with other use cases in mind may still find some parts of this page helpful.
+
+Numbered topics here assume correct verification of previous functionality so don't skip ahead.  You must establish correct operation for each numbered topic in turn or the deductions we make later may not be correct!  Each numbered topic also assumes the state and configuration established in previous topics.  For example, topic 3 instructs you to power on both your Rx and Tx module and verify the connection.  All subsequent topics assume they remain powered on and connected.
+
+Each numbered topic is structured as a test procedure with expected results followed by a list of things to check or tests to perform to help locate the trouble if the expected results are not observed.
+
+Note also that we don't mention things to check which were demonstrated to be correctly configured in previous topics.
```

Comment:
Yes, I was thinking about defining some terms.  I've tried to use Rx and Tx module for the radio modules and "serial rx" and "tx" for the signals.

Reply:
Most pages try and use the terms 'Tx module' and 'Receiver'

Reply:
For now, I've defined some terms and kept "Rx" instead of receiver because I kind of like the symmetry with Tx, but consistency is important.  Do you both think receiver is better?

Reply:
Is the 'Note' on the line necessary?  It seems like it is already mentioned in paragraph two above?

Reply:
Not explicitly I think, but implied.  Yes, perhaps I'm making this point more than needed.  I'll delete the note.